journal/modernized/08-chapter-7.adoc

```diff
@@ -1,10 +1,10 @@
-== Section VII.
+== Chapter VII
 
-Of the government, and Chancellor Boyle`'s clemency towards Friends,
-upon William Edmundson`'s petitioning them against G. Clapham`'s cruelty,
-then priest of Mount-melick.--Of his prophetic warning to the inhabitants of Londonderry;
-and conference afterwards with the bishop and mayor of that place;
-and the settling of meetings for Discipline in Ireland, by George Fox, in 1669.
+[.chapter-synopsis]
+* The Government, and Chancellor Boyle`'s Clemency Towards Friends Upon William Edmundson`'s Petitioning Them Against G. Clapham`'s Cruelty, Then Priest of Mount-melick
+* His Prophetic Warning To the Inhabitants of Londonderry; and Conference Afterwards With the Bishop and Mayor of That Place
+* The Settling of Meetings For Discipline In Ireland, By George Fox, In 1669
+* //I'm wondering if I should shorten these up at all? Not totally comfortable doing that yet.
```

Comment:
This is tough, agreed. If it makes you feel any better, I've never seen another book that needed this much massaging of the chapter-synopsis, so this is not very typical. Here's what I would propose (just a slight alteration from what you've got here):

```ASCIIDOC
[.chapter-synopsis]
* The government and Chancellor Boyle`'s clemency towards Friends, upon William Edmundson`'s petitioning them against G. Clapham`'s cruelty, then priest of Mount-melick
* Prophetic warning to the inhabitants of Londonderry
* Conference afterwards with the bishop and mayor of that place
* Settling of meetings for Discipline in Ireland, by George Fox, in 1669
```

You can copy/paste that block and use it for both editions.

Note that I didn't capitalize them -- I mentioned this in our slack DM yesterday but it was in the middle of a bunch of other stuff, so not sure if you saw it -- you don't need to keep capitalizing these, I realized I could do the capitalization programmatically as part of the computer's process of turning the asciidoc into pdfs and ebooks. So, there's no need to change back the old ones, but you can skip the step of capitalizing from here on out.

Reply:
Yes, i think I had done that before you wrote the message. I had a couple of things to finish up on that by looking at the PDF, so it wasn't sent right away.